app/apps/permits/api_queries_decos_join.py

```diff
@@ -299,7 +299,51 @@ def get_decos_entry_by_bag_id(self, bag_id, dt):
                                 if d.get("permit_type") == conf.get(
                                     DecosJoinConf.PERMIT_TYPE
                                 ):
-                                    d.update(permit_serializer.data)
+                                    permit_raw_data = d.get("raw_data", {})
+                                    # Check if there's already a permit with the same type AND raw data.
+                                    if permit_raw_data:
+                                        # Permit data found so check which one is valid now.
+                                        now = datetime.now().isoformat()
+                                        serializer_raw_data = (
+                                            permit_serializer.data.get("raw_data", {})
+                                        )
+                                        serializer_valid_from = serializer_raw_data.get(
+                                            "date6", None
+                                        )
```

Comment:
`data.get("key", None)` ==> de None is hierbij niet nodig. Als de key niet bestaat krijg je None terug van de get() method.